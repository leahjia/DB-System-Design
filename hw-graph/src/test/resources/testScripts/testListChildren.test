# University Map Children of Node testing file.
# Tests that a list of destinations from a node can be computed

CreateGraph map1
AddEdge map1 n1 n2 e1
AddEdge map1 n2 n1 e2

# Finds the children of node n1
ListChildren map1 n1

# Finds the children of node n2
ListChildren map1 n2