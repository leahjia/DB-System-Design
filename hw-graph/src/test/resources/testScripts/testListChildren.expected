# University Map Children of Node testing file.
# Tests that a list of destinations from a node can be computed

# Finds the children of node A
the children of parentNode in map1 are: Node B

# Finds the children of node B
the children of parentNode in map1 are: Node A
